layers/meta-opentrons/recipes-images/images/opentrons-ot3-image.bb

```diff
@@ -87,20 +89,27 @@ python do_create_opentrons_manifest() {
 ROOTFS_PREPROCESS_COMMAND += "do_create_opentrons_manifest; "
 
 # add the rootfs version to the welcome banner
-fakeroot do_add_rootfs_version() {
+do_add_rootfs_version() {
     printf "${DISTRO_NAME} ${DISTRO_VERSION} (${DISTRO_CODENAME}) \\\n \\\l\n" > ${IMAGE_ROOTFS}/etc/issue
     printf "${DISTRO_NAME} ${DISTRO_VERSION} (${DISTRO_CODENAME}) %%h\n" > ${IMAGE_ROOTFS}/etc/issue.net
     printf "${IMAGE_NAME}\n\n" >> ${IMAGE_ROOTFS}/etc/issue
     printf "${IMAGE_NAME}\n\n" >> ${IMAGE_ROOTFS}/etc/issue.net
 
+    # add datetime as version, example 202210111213
+    date +"%Y%m%d%H%M" > ${IMAGE_ROOTFS}/etc/version
+
     # add the VERSION.json file
     cat ${DEPLOY_DIR_IMAGE}/VERSION.json > ${IMAGE_ROOTFS}/etc/VERSION.json
 
-    # add hostname and machine-info
+    # add hostname to rootfs
+    printf "" > ${IMAGE_ROOTFS}/etc/hosts
     printf "opentrons" > ${IMAGE_ROOTFS}/etc/hostname
-    printf "PRETTY_HOSTNAME=opentrons\n" > "${IMAGE_ROOTFS}/etc/machine-info"
-    # TODO(ba, 2022-10-18): add proper mechanism for setting DEPLOYMENT
-    printf "DEPLOYMENT=development\n" >> "${IMAGE_ROOTFS}/etc/machine-info"
+    printf "PRETTY_HOSTNAME=opentrons\n" > ${IMAGE_ROOTFS}/etc/machine-info
+    printf "DEPLOYMENT=development\n" >> ${IMAGE_ROOTFS}/etc/machine-info
+
+    # cleanup
+    rm -rf ${IMAGE_ROOTFS}/opentrons_versions
+    rm -rf ${IMAGE_ROOTFS}/unit_tests
```

Comment:
what's adding these?

Reply:
removed /unit_tests dir, it was being added by imx-vpu-hantro
as for the opentrons_versions this temporarily holds the version json files to create the VERSION.json file.

Reply:
can we put the `unit_tests` dir in a bbappend for `imx-vpu-hantro`? want to avoid the image recipe kind of spiraling into a catchall for that stuff

Reply:
oops looks like you did! sorry!